pymecompress/quantize.c

```diff
@@ -36,7 +36,7 @@ void quantize_u16_noavx(uint16_t *data, uint8_t * out, int size, float offset, f
 
     for (i = 0; i < size; i++)
     {
-        out[i] = qs*sqrtf(data[i] - offset);
+        out[i] = round(qs*sqrtf(data[i] - offset));
```

Comment:
Should be `roundf()` not `round()`  - suggested modification will promote to double before rounding. While here,  we should probably explicitly cast the output to `unit8_t` to be safe.

Reply:
Good catch! Should be all set.